flags.go

```diff
@@ -195,9 +195,8 @@ func newApp() (app *cli.App) {
 			cli.DurationFlag{
 				Name:  "max-retry-sleep",
 				Value: time.Minute,
```

Comment:
Should we change the default Value to 30s here? Because earlier it was 30s and we haven't got any issues with that.